### PR TITLE
botmeta: cloudstack migrated to ngine_io.cloudstack

### PR DESCRIPTION
**Pull request by resmo**
##### SUMMARY
cloudstack related ansible content has been migrated to dedicated collection.

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME
cloudstack

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by felixfontein** [comment]
The [lib/ansible/config/routing.yml](https://github.com/ansible/ansible/blob/devel/lib/ansible/config/routing.yml) entries also need to be adjusted.

File: .github/BOTMETA.yml [hunk 1]
```diff
@@ -3356,7 +3356,7 @@ files:
   $module_utils/cloudstack.py:
     maintainers: $team_cloudstack
     labels: cloudstack
-    migrated_to: community.general
+    migrated_to: ngine_io.cloudstack
```
Comment (bcoca) [inline .github/BOTMETA.yml:3359]: you also need the per plugin changes here, not just in routing.yml

**Review by bcoca — DISMISSED** [review]

**Comment by gundalow** [comment]
@resmo Could you please rebase this? feel free to pull in https://github.com/ansible/ansible/pull/68916 into this PR if that's easier.

Once this is merged we can backport these three PRs to `stable-2.10 (I'm OK with a single PR)

https://github.com/ansible/ansible/pull/70095
https://github.com/ansible/ansible/pull/68917
https://github.com/ansible/ansible/pull/68916

**Comment by resmo** [comment]
@gundalow okay, you get an updated PR asap.

**Comment by felixfontein** [comment]
What's with the following modules?
- cloudscale_floating_ip
- cloudscale_server
- cloudscale_server_group
- cloudscale_volume

Also, what's with the inventory plugin `cloudscale`?

They have also been removed from community.general.

**Comment by resmo** [comment]
ready_for_review

**Review by felixfontein — APPROVED** [review]

**Comment by resmo** [comment]
> What's with the following modules?
> 
>     * cloudscale_floating_ip
> 
>     * cloudscale_server
> 
>     * cloudscale_server_group
> 
>     * cloudscale_volume
> 
> 
> Also, what's with the inventory plugin `cloudscale`?
> 
> They have also been removed from community.general.

this PR is about cloud**stack*,  not cloud**scale**

**Comment by gundalow** [comment]
rebuild_merge

**Pull request merged**